maintainer/travis/build_docker.sh

```diff
@@ -19,7 +19,7 @@ fi
 
 if [ "$TRAVIS_OS_NAME" = "linux" ]; then
 	image=espressomd/buildenv-espresso-$image:python
-	docker run -u espresso --env-file $ENV_FILE -v ${PWD}:/travis -it $image /bin/bash -c "git clone --recursive /travis && cd travis && maintainer/travis/build_cmake.sh"
+	docker run -u espresso --env-file $ENV_FILE -v ${PWD}:/travis -it $image /bin/bash -c "cp -r /travis .; cd travis && maintainer/travis/build_cmake.sh" || exit 1
```

Comment:
I had to switch this to `cp` because CentOS contains a Git version that does not support cloning shallow repositories (Travis appears to have switched to shallow some time ago). The `&&` needed to be replaced with `;` because there is a hidden file created by travis in there that cannot be copied.